src/h5web/vis-packs/core/utils.test.ts

```diff
@@ -92,58 +92,60 @@ describe('Shared visualization utilities', () => {
   });
 
   describe('extendDomain', () => {
-    it('should extend domain by factor', () => {
+    it('should extend domain with linear scale', () => {
       const extendedDomain = extendDomain([0, 100], 0.5);
       expect(extendedDomain).toEqual([-50, 150]);
     });
 
-    it('should return a non-empty domain when given a single positive value domain', () => {
-      const extendedDomain = extendDomain([1, 1], 0.5);
-      expect(extendedDomain).toEqual([0, 2]);
+    it('should extend domain with symlog scale', () => {
+      const [min, max] = [-10, 0];
+      const [extMin, extMax] = extendDomain([min, max], 0.1, ScaleType.SymLog);
+      expect(extMin).toBeLessThan(min);
+      expect(extMax).toBeGreaterThan(max);
     });
 
-    it('should return a non-empty domain when given a single negative value domain', () => {
-      const extendedDomain = extendDomain([-1, -1], 0.5);
-      expect(extendedDomain).toEqual([-2, 0]);
-    });
+    it('should extend domain with log scale', () => {
+      const [min, max] = [1, 10];
+      const [extMin1, extMax1] = extendDomain([min, max], 0.1, ScaleType.Log);
+      expect(extMin1).toBeLessThan(min);
+      expect(extMax1).toBeGreaterThan(max);
 
-    it('should extend domain by factor 1 with log scale', () => {
       // Extension factor of 1 for log scale means one decade
-      const [extMin, extMax] = extendDomain([10, 100], 1, ScaleType.Log);
-      expect(extMin).toBeCloseTo(1);
-      expect(extMax).toBeCloseTo(1000);
+      const [extMin2, extMax2] = extendDomain([10, 100], 1, ScaleType.Log);
+      expect(extMin2).toBeCloseTo(1);
+      expect(extMax2).toBeCloseTo(1000);
     });
 
-    it('should extend domain by factor with log scale', () => {
-      const [min, max] = [1, 10];
-      const [extMin, extMax] = extendDomain([min, max], 0.1, ScaleType.Log);
-      expect(extMin).toBeLessThan(min);
-      expect(extMax).toBeGreaterThan(max);
+    it('should extend positive single-value domain', () => {
+      const extendedDomain = extendDomain([1, 1], 0.5);
+      expect(extendedDomain).toEqual([0, 2]);
```

Comment:
Could you add a similar test for `ScaleType.Log` ?

Reply:
Hmm I found another edge case... 

```ts
const extendedDomain = extendDomain([1, 1], 1, ScaleType.Log); // => `[0, +Infinity]` => `[0, 1 / Number.EPSILON]`
```

Reply:
Yeah, so we'd return `Number.EPSILON` in this case?

Reply:
Seems weird to return the largest possible log domain when passing `[1, 1]`, though.